File: Database/MySQL/MySQL.py
#MySQL
import mysql.connector as msc

connection = msc.connect(
    host='localhost',
    user='root',
    password='sudeep23mysq',
    database='demo'
)
cursor = connection.cursor()

cursor.execute('create table data (name varchar(255), iq varchar(255))')

cursor.execute('insert into data (name, iq) values (%s, %s)', ('aarush', '90'))
cursor.execute('insert into data (name, iq) values (%s, %s)', ('aryan', '60'))
cursor.execute('insert into data (name, iq) values (%s, %s)', ('shivam', '-30'))

cursor.execute('update data set iq=%s where iq<%s', ('30', '0'))
cursor.execute('delete from data where iq=%s', ('60',))

cursor.execute('select * from data')
res = cursor.fetchall()
for i in res :
    print(i)

connection.commit()
cursor.close()
connection.close()

# pip install mysql-connector-python

File: Database/MySQL/MySQL.js
//MySQL
import mysql from 'mysql2/promise';

let func = async () => {
    const connection = await mysql.createConnection({
        host: 'localhost',
        user: 'root',
        password: 'sudeep23mysq',
        database: 'demo'
    });

    await connection.execute('create table data (name varchar(255), iq varchar(255))');

    await connection.execute('insert into data (name, iq) values (?, ?)', ['aarush', 90]);
    await connection.execute('insert into data (name, iq) values (?, ?)', ['aryan', 60]);
    await connection.execute('insert into data (name, iq) values (?, ?)', ['shivam', -30]);

    await connection.execute('update data set iq=? where iq<?', [30, 0]);
    await connection.execute('delete from data where iq=?', [60]);

    const [res] = await connection.execute('select * from data');
    for (let i=0;i<res.length;i++)
        console.log(res[i]);

    await connection.end();
}
func();

// npm install mysql2